CMakeLists.txt

```diff
@@ -649,7 +649,12 @@ if(testing)
   endif()
 endif()
 
-cmake_host_system_information(RESULT PROCESSOR QUERY PROCESSOR_DESCRIPTION)
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.10.0")
+  if (${CMAKE_MINIMUM_REQUIRED_VERSION} VERSION_GREATER_EQUAL "3.10.0")
+    message(FATAL_ERROR "Remove this condition")
```

Comment:
This breaks the compilation for me, when a dependency is pulled in, which sets a higher minimum required cmake version. Why is this check and the fatal error needed? Wouldn't it be sufficient to just check `
if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.10.0")`